sdks/python/setup.py

```diff
@@ -137,11 +137,14 @@ def get_version():
     ]
 
 GCP_REQUIREMENTS = [
+    'cachetools>=3.1.0,<4',
     # google-apitools 0.5.23 and above has important Python 3 supports.
     'google-apitools>=0.5.26,<0.5.27',
-    'proto-google-cloud-datastore-v1>=0.90.0,<=0.90.4',
-    # [BEAM-4543] Datastore IO is not supported in Python 3.
+    # [BEAM-4543] googledatastore is not supported in Python 3.
```

Comment:
Can both google-cloud-datastore and googledatastore installed in the same env without conflicts ?

Reply:
Yes, having both installed is a large contributor to the complexity of this PR.
You can't import both in the same process.